src/generic/graphicc.cpp

```diff
@@ -867,6 +913,20 @@ wxCairoPenData::wxCairoPenData( wxGraphicsRenderer* renderer, const wxGraphicsPe
         }
         break;
     }
+
+    if (info.GetGradientType() == wxGRADIENT_LINEAR)
```

Comment:
Let's use `switch ( info.GetGradientType() )`, like this (some) compiler(s) will warn us if we add new enum elements, but forget to update the switch.